woocommerce/woocommerce-android
# Show card payment option on payment methods based on cpp eligibility

Closes: #6640 
Closes: #6638


### Description

1. Renames `TakePaymentFragment` to `SelectPaymentMethodFragment`
2. Show the "collect payment" button for all unpaid orders
3. Hide card payment method in `SelectPaymentMethodFragment` based on `CardReaderPaymentCollectibilityChecker`

### Testing instructions

* Try to collect payment for different orders/different statuses
* Notice that "collect payment" button visible for all unpaid orders
* Notice card payment method is visible only by the logic determined in `CardReaderPaymentCollectibilityChecker`

### Images/gif


https://user-images.githubusercontent.com/4923871/171589597-246e7ad7-2ec8-4100-bd50-568f36a68683.mp4
